cmd/vulkan_sample/vulkan_sample.exports

```diff
@@ -0,0 +1,2 @@
+ANativeActivity_onCreate
```

Comment:
Looks like we don't have this entry point

Reply:
This is a special one defined by android_native_app_glue, which is provided by the NDK (see https://android.googlesource.com/platform/ndk.git/+/refs/heads/master/sources/android/native_app_glue/android_native_app_glue.c#424 )